grafana-plugin/src/models/outgoing_webhook/outgoing_webhook.ts

```diff
@@ -92,6 +95,19 @@ export class OutgoingWebhookStore extends BaseStore {
     });
   }
 
+  @action.bound
+  @AutoLoadingState(ActionKey.TRIGGER_MANUAL_WEBHOOK)
+  @WithGlobalNotification({ success: 'Webhook has been triggered successfully.', failure: 'Failed to trigger webhook' })
+  async triggerManualWebhook(id: ApiSchemas['Webhook']['id'], alertGroupId: ApiSchemas['AlertGroup']['pk']) {
+    await makeRequest(`${this.path}${id}/trigger_manual`, {
```

Comment:
if it's new endpoint, is it possible to expose OpenAPI schema right away and use typed httpClient @matiasb @teodosii ?

Reply:
We don't have schemas setup for webhook endpoints atm, but I can take a look.